react/src/components/VFolderNodes.tsx

```diff
@@ -230,22 +233,34 @@ const VFolderNodes: React.FC<VFolderNodesProps> = ({
                   )}
                   {/* Restore */}
                   {isDeletedCategory(vfolder?.status) && (
-                    <Tooltip title={t('data.folders.Restore')} placement="left">
+                    <Tooltip
+                      title={
+                        isPipelineFolder
+                          ? t('data.folders.CannotRestorePipelineFolder')
```

Comment:
Pipeline folders will be permanently deleted without going through the trash bin. However, if a folder is already in the trash bin, I don't think it needs to be disabled from being restored.

Reply:
I have some questions :)

Is the Fasttrack always using the new version? I've also disabled the restore button to ensure backwards compatibility.

Also, I know that pipeline is a folder that cannot be created arbitrarily, but is automatically created when creating a pipeline, is it okay to make the pipeline folder restoreable?

Reply:
1. The primary policy is to align the version with the core.
2. You are right. As the pipeline folder can be deleted by the pipeline only, it would be better to disable restoration from WebUI.